contracts/interfaces/IForwarder.sol

```diff
@@ -50,6 +50,11 @@ interface IForwarder {
         bytes calldata signature
     ) external view;
 
+     /**
+     * returns the encrypted owner that deployed smart wallet.
+     */
+    function getOwner() external view returns (bytes32 owner);
```

Comment:
This change is already included in the ethers branch, so I just include it, here.

Reply:
not public, but we need to change its accessibility so I decided to do the same change that we applied before.